pkg/tfvars/vsphere/vsphere.go

```diff
@@ -7,18 +7,19 @@ import (
 )
 
 type config struct {
-	VSphereURL      string `json:"vsphere_url"`
-	VSphereUsername string `json:"vsphere_username"`
-	VSpherePassword string `json:"vsphere_password"`
-	MemoryMiB       int64  `json:"control_plane_memory_mib"`
-	DiskGiB         int32  `json:"control_plane_disk_gib"`
-	NumCPUs         int32  `json:"control_plane_num_cpus"`
-	Cluster         string `json:"vsphere_cluster"`
-	Datacenter      string `json:"vsphere_datacenter"`
-	Datastore       string `json:"vsphere_datastore"`
-	Folder          string `json:"vsphere_folder"`
-	Network         string `json:"vsphere_network"`
-	Template        string `json:"vsphere_template"`
+	VSphereURL        string `json:"vsphere_url"`
+	VSphereUsername   string `json:"vsphere_username"`
+	VSpherePassword   string `json:"vsphere_password"`
+	MemoryMiB         int64  `json:"control_plane_memory_mib"`
+	DiskGiB           int32  `json:"control_plane_disk_gib"`
+	NumCPUs           int32  `json:"control_plane_num_cpus"`
```

Comment:
Do we need the s in NumCPUs ??

Reply:
> Do we need the s in NumCPUs ??

I think we should keep it this way to be consistent with the machine-api https://github.com/openshift/machine-api-operator/blob/master/pkg/apis/vsphereprovider/v1alpha1/vsphereproviderconfig_types.go#L38